src/index.tsx

```diff
@@ -532,6 +545,49 @@ export function useCamera() {
     return () => eventListener.remove();
   }, []);
 
+  /**
+   * sets track encoding that server should send to the client library.
+   * The encoding will be sent whenever it is available. If choosen encoding is
+   * temporarily unavailable, some other encoding will be sent until choosen encoding
+   *  becomes active again.
+   *
+   * @param trackId id of a track which encoding you want to select
+   * @param encoding encoding to select
+   */
+  const setTargetTrackEncoding = useCallback(
```

Comment:
this one is not related to the local video track, so it would be better to move it to separate function?

Reply:
I placed it in a new hook.